Resources/doc/reference/configuration/headers.rst

```diff
@@ -218,7 +243,12 @@ This value must be a valid input to ``DateTime``.
                     headers:
                         last_modified: "-1 hour"
 
-.. hint::
+.. note::
+
+    Do **not** use last modified and the ETag together, as this only generates
+    confusion. In most situations, you should prefer the ETag method. Last
```

Comment:
This is very opiniated. Some sources (will try to find them later) consider setting both a best practice.